vendor/vtd/conanfile.py

```diff
@@ -43,17 +46,48 @@ def configure(self):
     def build(self):
         src = Path(self.source_folder)
         dst = Path(self.build_folder)
+        vtddir = dst / "VTD.2.2"
+        libdir = vtddir / "lib"
 
         def extract_archive(archive):
             print(f"Extracting: {archive}")
             tools.untargz(src / archive, dst)
 
         extract_archive(self._archive_base)
+        libdir.mkdir()
         if self.options.with_osi:
             extract_archive(self._archive_osi)
         if self.options.with_road_designer:
             extract_archive(self._archive_rod)
 
+        # Get a list of all binaries we need to set the RPATH of:
+        result = subprocess.run(
+            """find -type f -executable -exec sh -c "file -i '{}' | grep -q '; charset=binary'" \; -print""",
+            shell=True,
+            stdout=subprocess.PIPE,
+            check=True,
+        )
+        binary_files = [Path(x) for x in result.stdout.decode().splitlines()]
+        # Set RPATH of all the collected binaries:
+        for file in binary_files:
+            assert file.exists()
```

Comment:
Here one could add 
```
            if not ".so" in file.suffixes:
                continue
```
to avoid a bunch of error messages when it tries to set RPATH e.g. on pdf files.